### PR TITLE
TRIVIAL: Changelog for 5.2.0 release

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/882/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/418/ : SUCCESS in 3m 49s

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
 # Changelog
+## 5.2.0 (August 10, 2018)
+
+- Added Bubble chart ([commit](https://github.com/gooddata/gooddata-react-components/commit/2e4031b9296275e60a4c90b171fa7c2064c0e11e))
+- Added Treemap chart ([commit](https://github.com/gooddata/gooddata-react-components/commit/05a77cc4041680468290e7af8eb089698f2afd69))
+- Change way how null values are rendered in Area chart ([commit](https://github.com/gooddata/gooddata-react-components/commit/65e923ed9ed6d23c3c0530b2e95df5beca7f1aa9))
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: Changed the way how Area chart renders null values

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+- Time over time comparison ([commit](https://github.com/gooddata/gooddata-react-components/pull/424/commits/a54ec123589809f71c6aa00e534f099b1bbf06e2))
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: Added the time over time comparison feature

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+- Second measure in headline component ([commit](https://github.com/gooddata/gooddata-react-components/commit/15aa8a17cad493c2a9b2651c7622e9df878b13e8))
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: Added the option to add the second measure to headlines

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/884/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/420/ : SUCCESS in 2m 40s

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/171/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/427/ : SUCCESS in 1m 37s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/235/ : UNSTABLE in 9m 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/66/ : SUCCESS in 14m 17s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/172/ : SUCCESS in 25s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/428/ : SUCCESS in 2m 34s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/236/ : SUCCESS in 10m 42s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/67/ : SUCCESS in 9m 58s

**Pull request merged**